### ISSUE TITLE
Is there a way to make task id simple positive integer?

### ISSUE THREAD

**Issue by ataran17** [mkloubert/vscode-kanban#17]
Hello.
Currently task id is big string like '20180702151728_179455396_9237fae6e510bfd542e5cba517105482' which is fine to be too unique.
It would be great having it as simple positive integer like 1,2,3... or add another field with autoincrement(+1).
Thus it can be put in commit like 'closing task id #1' in clear way.

**Comment by HamzaAnis** [comment]
Yes a small integer task Id would be good but currently Id has itself some other information in it which is needed to identify the card details so it will be better to add another field to search the card.

**Comment by mkloubert** [comment]
@ataran17 @HamzaAnis 

Sorry for the late answer, but **I had now time to implement the feature in version 1.22.0**.

There is a new [setting](https://github.com/mkloubert/vscode-kanban#settings-), called `simpleIDs`, which indicates to use simple integer values as IDs and is `(true)` by default.

There is not a bug, but a small "issue":

If you have existing cards, which have IDs like `20180531184916_11696734_e0f09d02db244a4db9409fe5b53e7bc1`, you should remove all `id` properties from all entries in your `.vscode/vscode-kanban.json` file and save your board manually (by "save" button the upper right corner of the WebView), otherwise the next ID, that will be used is something like `20180531184917`.

If you create a new board, that "issue" will not appear.

**Comment by HamzaAnis** [comment]
Great. Good to know that. This issue can be closed or whatever you want.
Do you have plans to improve the UI for it?

**Comment by mkloubert** [comment]
@HamzaAnis 

Do you mean a better look?

**Comment by HamzaAnis** [comment]
Yes.

**Comment by ataran17** [comment]
I think it can be shown now along with the task name on board (in case it is integer).
So that one could copy/paste it right from board task to commit field.
